src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -213,7 +213,7 @@ <h5 class="col-sm-2 margin-top-0">
                 <div class="col-sm-10">
                   <div class="panel panel-default panel-body mce-content-body content-editor" id="instructions" spellcheck="false">
                     <p>
-                      updated instructions
+                      Please give your feedback based on the following questions.
```

Comment:
I don't understand why everytime this line cause builds failure.
If all the CI test got passed in the Godmode then how come this build is failing :thinking: